.github/workflows/ci.yaml

```diff
@@ -244,7 +264,7 @@ jobs:
 
       - name: Test
         run: |
-          git submodule sync
+          git submodule sync && cat .gitmodules
```

Comment:
Why only here?

Reply:
Oops, forgot to remove that while debugging